src/Struct/AdyenPaymentMethodDataStruct.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <shopware@adyen.com>
+ */
+
+namespace Adyen\Shopware\Struct;
+
+use Shopware\Core\Framework\Struct\Struct;
+
+class AdyenPaymentMethodDataStruct extends Struct
+{
+    /**
+     * @var bool
+     */
+    protected $isAdyen = false;
```

Comment:
@peterojo I think we could actually remove this flag because I could just not add the extension at all for non adyen payment methods. However, this way one can be sure that it was not a but that there is not adyen data added to the payment method.

Reply:
@peterojo removed it now.